README.md

```diff
@@ -1 +1,23 @@
 # hpqtypes-effectful
+
+## Description
+
+`effectful` bindings for the `hpqtypes` haskell bindings for `libpqtypes`
+
+## How to use
+
+This library provides an EffectDB effect that allows use of the hpqtypes bindings for libpqtypes in the effectful ecosystem."
```

Comment:
let's get rid of " at the end.

also some language nit-picking from a non-native speaker so feel free to ignore me: "allows use of" ->"allows the use of"?

Reply:
My totally non-native (and usually wrong) opinion is "allows use of" sounds nicer, but a quick search only brings up news titles of the form "X allows use of Y" (e.g. [here](https://www.theepochtimes.com/quebecs-new-academic-freedom-bill-allows-use-of-any-word-in-university-academic-context_4392004.html)).